ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -260,6 +295,18 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-sno-lvms
+- as: aws-ipi-usertags-custom-sg-arm-f28
```

Comment:
Seem like this one is duplicated with the following one `aws-ipi-usertags-custom-sg-fips-amd`

Reply:
removed